### PR TITLE
Adding 3.2 proxy updates repo

### PR DESCRIPTION
**Pull request by Bischoff**
Note: I did not find how to add it to minima.yaml. Maybe it's not needed. Could you please check?

### CODE REVIEW

File: salt/repos/suse_manager_proxy.sls [hunk 1]
```diff
@@ -35,8 +35,11 @@ suse_manager_proxy_pool_repo:
     - source: salt://repos/repos.d/SUSE-Manager-Proxy-3.2-x86_64-Pool.repo
     - template: jinja
 
-# NOTE: the update repository for 3.2 does not exist yet
-#       don't forget to add it after GA
```
Comment (juliogonzalez) [inline salt/repos/suse_manager_proxy.sls]: Next time we should create an issue for this! :-D

Comment (juliogonzalez) [reply salt/repos/suse_manager_proxy.sls]: I was not blaming, just suggesting an improvement for next time :-D

What I usually do to be 100% sure I do not forget about it, is assigning those things to myself, so I can ping people (even if I'm not going to do it).

That's why you can see me as assigned at some strange issues, such as some bugs.

Comment (Bischoff) [reply salt/repos/suse_manager_proxy.sls]: Yes, that's a technique too.

**Comment by moio** [comment]
`minima.yaml` already contains the channel

https://github.com/moio/sumaform/blob/master/salt/mirror/minima.yaml#L57

**Comment by moio** [comment]
Thanks @Bischoff !!!

**Comment by Bischoff** [comment]
> Please note SUSE Manager 3.2 Server Updates is also missing

Rats... :rat::rat::rat: OK, I will do it in a separate PR, or you do it. Whoever comes first and does not forget.

**Pull request merged**